Guide_1_Send_Payment.md

```diff
@@ -50,23 +68,67 @@ A detailed description of each field of the payment resource is available in the
 
 ### Create the Submission Resource
 
-The next step is to send the payment by creating a payment submission resource. Note that you have to provide the payment ID in the call.
-
 ```json
 Code for submission resource here!
 ```
 
+The next step is to send the payment by creating a payment submission resource. Note that you have to provide the payment ID in the call.
 
+And just like that, your payment is on its way!
 
 ## Track the Payment Manually
 
-...
+_**(Question: Should Audits be mentioned here? Since it said in the email that audits are not recommended and this is a getting started guide, maybe it's more straightforward and less confusing not to mention it?)**_
 
+There are several ways to monitor your payment and track it on its way through the system. The easiest one is to query the submission resource that you created above:
 
+`GET /transaction/payments/{payment_id}/submissions/{submission_id}`
+
+Note that the submission resource is identified using the `payment_id` and the `submissions_id`.
+
+The response contains the current status of the payment in an attribute called `status`. If the payment has been successful, the status attribute says `delivery_confirmed`. Failed payments are denoted with a status attribute value `delivery_failed`. In this case, the attribute `status_reason` contains further information about why the payment failed. 
+
+A detailed description of all possible values of the status attribute and their meaning is available in the [API documentation](http://draft-api-docs.form3.tech/?http#payment-submission-status).
 
 ## Track the Payment using Subscriptions
 
-...
+Another way of tracking your payment is to use subscriptions. You can subscribe to certain events and automatically be notified when they occur. 
+
+The Form3 API supports [Amazon SQS](https://aws.amazon.com/sqs/) and webhook URLs to notify subscribers when an event occurs.
+
+**_(Note: I've used Slate HTML syntax below, although you can't see it on GitHub because they strip the HTML before displaying. We'll use Slate to display this guide, right?)_**
+
+<aside class="notice">
+Using Amazon SQS is recommended, since it is a managed, highly-available solution and less error-prone than webhooks. For the sake of simplicity, however, this guide uses a webhook to demonstrate the subscription API.</aside>
+
+### Create a Subscription using Webhooks
+
+In order to subscribe to an event via webhook, you need to provide the API with a callback URL. This needs to be a public URL that the API can call and notify you whenever the event has occured.
+
+### Using ngrok to create a public URL
```

Comment:
Can you try using requestbin instead of ngrok? I think that would make this section simpler to set up and test.